packages/ngspice/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: ngspice
+  version: "44.2"
+
+source:
+  url: https://downloads.sourceforge.net/project/ngspice/ng-spice-rework/44.2/ngspice-44.2.tar.gz
+  sha256: e7dadfb7bd5474fd22409c1e5a67acdec19f77e597df68e17c5549bc1390d7fd
+
+build:
+  type: shared_library
+  script: |
+    ./autogen.sh
+
+    configure_args=(
```

Comment:
You'll probably set CFLAGs and LDFLAGs for Pyodide to generate proper shared library.

See libmagic example.

https://github.com/pyodide/pyodide/blob/4fbbbedc09496c6968086d69aadba75398718b13/packages/libmagic/meta.yaml#L23-L27

Reply:
Yea I tried that, and just committed those changes, but I still get the same error. I'm not sure how to debug this.

Reply:
I've been fiddling more with the ngspice build system but no progress.